1. What is the use of try & catch?
a) It is used to manually handle the exception
b) It helps to fix the errors
c) It prevents automatic terminating of the program in cases when an exception occurs
d) All of the mentioned
d
2. What will be the output of the following C# code?
 class Output  {     public static void main(String args[])      {          try          {             int a = 9;             int b = 5;             int c = a / b - 5;             Console.WriteLine("Hello");         }         catch(Exception e)          {             Console.WriteLine("C");         }          finally          {             Console.WriteLine("sharp");         }      } }
a) Hello
b) C
c) Hellosharp
d) Csharp
d
3. Choose the statement which is incorrect?
a) try block does not need to be followed by catch block
b) try block can be followed by finally block instead of catch block
c) try can be followed by both catch and finally block
d) try need not to be followed by anything
d
4. What will be the output of the following C# code?
  class Output   {      public static void main(String args[])       {          try           {              int a = 10;              int b = 5;              int c = a / b - 5;              Console.WriteLine("Hi");          }          catch(Exception e)           {              Console.WriteLine("hello");          }       }  }
a) Hi
b) hello
c) Hihello
d) Compile time error
b
5. Which of the keywords are used for the block to be examined for exceptions?
a) try
b) catch
c) throw
d) check
a
6. Which of these keywords are used for the block to handle the exceptions generated by try block?
a) try
b) catch
c) throw
d) check
b
7. What will be the output of the following C# code?
  class Output   {      public static void main(String args[])       {         try          {             int a = 5;             int b = 10;             int c = b / a - 5;             Console.WriteLine("Csharp");         }      } }
a) Csharp
b) sharp
c) C
d) Compile time error
d
8. What will be the output of the following C# code snippet?
 class Output  {     public static void main(String args[])     {        try         {            int a = 0;            int b = 5;            int c = a / b - 5;            Console.WriteLine("C");        }        finally         {            Console.WriteLine("sharp");        }     }}
a) C
b) sharp
c) Csharp
d) None of the mentioned
c
9. What will be the output of the following C# code snippet?
 class Output  {     public static void main(String args[])      {         try          {             int a = 10;             int b = 5;             int c =  b - 5 / 5;             Console.WriteLine("Hi");         }         catch(Exception e)          {             Console.WriteLine("hello");         }      } }
a) Hi
b) hello
c) Hihello
d) Compile time error
a
10. Which of these keywords are used for generating an exception manually?
a) try
b) catch
c) throw
d) check
c
Sanfoundry Global Education & Learning Series – C# Programming Language.
C# Practice ResourcesC# Mock Tests & Certification Test | 1000 C# MCQs | 1000 C# Programs | Best C# Books
Participate in the Sanfoundry Certification contest to get free Certificate of Merit. Join our social networks below and stay updated with latest contests, videos, internships and jobs!
Telegram | Youtube | LinkedIn | Instagram | Facebook | Twitter | Pinterest
Youtube | LinkedIn | Instagram | Facebook | Twitter | Pinterest
« Prev - C#Questions & Answers – Exceptions of Type Finally and Built in Exceptions» Next - C# Questions & Answers – Attributes 
